core/uploadDiveLogsDE.cpp

```diff
@@ -171,7 +178,7 @@ bool uploadDiveLogsDE::prepareDives(const QString &tempfile, bool selected)
 		snprintf(filename, PATH_MAX, "%d.xml", i + 1);
 		s = zip_source_buffer(zip, membuf, streamsize, 1); // frees membuffer!
 		if (s) {
-			int64_t ret = zip_add(zip, filename, s);
+                        int64_t ret = zip_file_add(zip, filename, s, ZIP_FL_OVERWRITE);
```

Comment:
Yet another unwanted whitespace change - please consider setting up your editor to apply the correct type of whitespace.